src/sizzle.js

```diff
@@ -1245,14 +1305,18 @@ Expr = Sizzle.selectors = {
 
 				result += "";
 
+				/* eslint-disable max-len */
```

Comment:
I think the following `return` expression was clearer originally without all this extra intendation. Can you revert that to the original indentation? If ESLint complains, you can add an override like you did in this line - and maybe remove the `max-len` one.

Reply:
I need to maintain the `max-len` rule. I hope the indentation is better now.